resources/qml/RoomDirectory.qml

```diff
@@ -180,6 +180,18 @@ ApplicationWindow {
         width: parent.width
         implicitHeight: roomSearch.height
 
+        MatrixTextField {
+            id: chooseServer
+
+            Layout.fillWidth: true
```

Comment:
Can you make it 30% of the window width and move it to the right? Ideally we would have a drop down with past selections, but I'd say this is a good start.

Reply:
Sure thing. You mean to the right of the search query text field, right?

Reply:
Please, see how it looks like now:
https://people.debian.org/~patryk/tmp/nheko/screenshots/choose_server_custom_v2.png